From: Kel Modderman <kel@otaku42.de>
Subject: Use full executable path into wpa_gui.desktop

Debian specific patch to desktop meny entry, so that we may exec
wpa_gui which being in /usr/sbin may not be in the PATH
---
--- a/wpa_supplicant/wpa_gui-qt4/wpa_gui.desktop
+++ b/wpa_supplicant/wpa_gui-qt4/wpa_gui.desktop
@@ -2,7 +2,7 @@
 Version=1.0
 Name=wpa_gui
 Comment=Graphical user interface for wpa_supplicant
-Exec=wpa_gui
+Exec=/usr/sbin/wpa_gui
 Icon=wpa_gui
 GenericName=wpa_supplicant user interface
 Terminal=false
